nana/modules/youtube.py

```diff
@@ -26,70 +26,55 @@
 
 ──「 **Convert to music** 」──
 -> `ytmusic (url)`
--> `ytaudio (url)`
 Download youtube music, and then send to tg as music.
 """
 
 
-@app.on_message(Filters.user(AdminSettings) & Filters.command(["ytdl"], Command))
+@app.on_message(filters.user(AdminSettings) & filters.command("ytdl", Command))
 async def youtube_download(_client, message):
 	args = message.text.split(None, 2)
-	await msg(message, text="Checking")
+	await edrep(message, text="Checking")
 	if len(args) == 1:
-		await msg(message, text="Write any args here!")
+		await edrep(message, text="Write any args here!")
 		return
 	try:
 		yt = YouTube(args[1])
 	except ValueError:
-		await msg(message, text="Invalid URL!")
+		await edrep(message, text="Invalid URL!")
 		return
 
 	if len(args) == 2:
 		link = args[1]
 		text = "🎬 **Title:** [{}]({})\n".format(escape_markdown(yt.title), link)
-		status = "**Downloading video...**\n"
-		await msg(message, text=status + text, disable_web_page_preview=True)
 		YouTube(link).streams.first().download('nana/downloads', filename="tempvid")
-		status = "**Uploading File To Telegram...**\n"
-		await msg(message, text=status + text, disable_web_page_preview=True)
 		await app.send_video(message.chat.id, video="nana/downloads/tempvid.mp4")
-		status = "**Removing Temp File...**"
-		await msg(message, text=status)
 		os.remove('nana/downloads/tempvid.mp4')
-		status = "** Done ✔️✔️**\n"
-		await msg(message, text=status + text, disable_web_page_preview=True)
+		await edrep(message, text=text, disable_web_page_preview=True)
 
 		return
 	if len(args) == 3:
 		link = args[1]
 		reso = args[2]
 		text = "🎬 **Title:** [{}]({})\n".format(escape_markdown(yt.title), link)
-		status = "**Downloading video...**\n"
-		await msg(message, text=status + text, disable_web_page_preview=True)
 		stream = yt.streams.filter(file_extension='mp4').filter(resolution="{}".format(reso)).first()
 		stream.download('nana/downloads', filename="tempvid")
-		status = "**Uploading File To Telegram...**\n"
-		await msg(message, text=status + text, disable_web_page_preview=True)
 		await app.send_video(message.chat.id, video="nana/downloads/tempvid.mp4")
-		status = "**Removing Temp File...**"
-		await msg(message, text=status)
 		os.remove('nana/downloads/tempvid.mp4')
-		status = "**Done ✔️✔️**\n"
-		await msg(message, text=status + text, disable_web_page_preview=True)
+		await edrep(message, text=text, disable_web_page_preview=True)
 		return
 
 
-@app.on_message(Filters.user(AdminSettings) & Filters.command(["ytmusic", "ytaudio"], Command))
+@app.on_message(filters.user(AdminSettings) & filters.command("ytmusic", Command))
```

Comment:
Codacy found an issue: [youtube_music is too complex (27) (MC0001)](https://app.codacy.com/gh/MadeByThePinsHub/Nana-Remix/pullRequest?prid=6330175)